frontend/app_flowy/lib/plugins/grid/presentation/widgets/header/field_cell_action_sheet.dart

```diff
@@ -31,38 +32,39 @@ class _GridFieldCellActionSheetState extends State<GridFieldCellActionSheet> {
   Widget build(BuildContext context) {
     if (_showFieldEditor) {
       final field = widget.cellContext.field;
-      return FieldEditor(
-        gridId: widget.cellContext.gridId,
-        fieldName: field.name,
-        typeOptionLoader: FieldTypeOptionLoader(
+      return SizedBox(
+        width: 400,
+        child: FieldEditor(
           gridId: widget.cellContext.gridId,
-          field: field,
+          fieldName: field.name,
+          typeOptionLoader: FieldTypeOptionLoader(
+            gridId: widget.cellContext.gridId,
+            field: field,
+          ),
         ),
       );
     }
-    return BlocProvider(
-      create: (context) =>
-          getIt<FieldActionSheetBloc>(param1: widget.cellContext),
-      child: IntrinsicWidth(
-        child: IntrinsicHeight(
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              _EditFieldButton(
-                cellContext: widget.cellContext,
-                onTap: () {
-                  setState(() {
-                    _showFieldEditor = true;
-                  });
-                },
-              ),
-              VSpace(GridSize.typeOptionSeparatorHeight),
-              _FieldOperationList(widget.cellContext),
-            ],
-          ),
+    return SizedBox(
+      width: 200,
```

Comment:
If we set the width to 200, a fixed width. then It can not auto-expand to the text width. For example
![image](https://user-images.githubusercontent.com/86001920/212214559-5fc6505b-8d59-4e5c-afe3-00c4a587054f.png)